src/test/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/MtAmazonDynamoDbStreamsBySharedTableTest.java

```diff
@@ -167,7 +167,7 @@ void testLimit() {
             putTestItem(mtDynamoDb, TENANTS[1], i++);
             // one record for tenant 1 on page 2 (don't expect to get)
             putTestItem(mtDynamoDb, TENANTS[1], i++);
-            final MtRecord expected2 = putTestItem(mtDynamoDb, TENANTS[0], i++);
+            putTestItem(mtDynamoDb, TENANTS[0], i++);
```

Comment:
Unused declaration

Reply:
should instead this `expected2` actually be used in checking something?

Reply:
I think the point of this line and the one above and below is to insert 3 records on the next page.  It doesn't matter what it's in them.